components/feature/webcompat/src/main/assets/extensions/webcompat/lib/shims.js

```diff
@@ -344,8 +344,10 @@ class Shims {
         }
       };
       browser.tabs.onRemoved.addListener(unmarkShimsActive);
-      browser.tabs.onUpdated.addListener(unmarkShimsActive, {
-        properties: ["discarded", "url"],
+      browser.tabs.onUpdated.addListener((tabId, changeInfo) => {
```

Comment:
@denschub, I see on the [docs](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/onUpdated) that `browser.tabs.onUpdated.addListener` supports both `changeInfo` and `properties` parameters. Did you find that `properties` was not working for you? Should we file a bug for the GeckoView team to take a look? 

<img width="710" alt="image" src="https://user-images.githubusercontent.com/773158/120241971-d8b72780-c231-11eb-882a-24bbc5ecd4e7.png">

Reply:
Ah yeah, I didn't actually create the patch, I was just landing it for Tom who took a break. :) See his comment below, https://github.com/mozilla-mobile/android-components/pull/10390#issuecomment-851706028

Reply:
👍🏽